### ISSUE TITLE
Structural tags missing in PDF

### ISSUE THREAD

**Issue by dannychapman** [department-of-veterans-affairs/vets-website#860]
https://www.vets.gov/playbook/downloads/Toward_A_Veteran_Centered_VA.pdf
